ironic/resource_ironic_node_v1.go

```diff
@@ -457,6 +457,24 @@ func schemaToCreateOpts(d *schema.ResourceData) *nodes.CreateOpts {
 	}
 }
 
+// UpdateNode wraps gophercloud's update function, so we are able to retry on 409 when Ironic is busy.
+func UpdateNode(client *gophercloud.ServiceClient, uuid string, opts nodes.UpdateOpts) (*nodes.Node, error) {
+	interval := 5 * time.Second
+	for retries := 0; retries < 5; retries++ {
+		node, err := nodes.Update(client, uuid, opts).Extract()
+		switch err.(type) {
+		case gophercloud.ErrDefault409:
+			log.Printf("[DEBUG] Failed to update node: ironic is busy, will try again in %d seconds", interval)
+			time.Sleep(interval)
+			interval *= 2
```

Comment:
This doubles each time, so we would wait up to 5 retries: 5, 10, 20, 40, 80 seconds - a bit under 3 minutes in total, which is probably more generous than it needs to be.